app/src/main/java/com/example/whatsapp_clone/Model/Retrofit/CreateChatPOJO.java

```diff
@@ -3,7 +3,7 @@
 import com.example.whatsapp_clone.Model.User;
 import com.google.gson.annotations.SerializedName;
 
-public class ChatMetadata {
+public class CreateChatPOJO {
```

Comment:
POJO?

Reply:
https://www.scaler.com/topics/pojo-class-in-java/#